tests/end_to_end/selenium_test.py

```diff
@@ -27,15 +29,18 @@ def test_app():
     thread.join(1)
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture(scope='session')
 def driver():
     opts = webdriver.ChromeOptions()
     opts.add_argument('--headless')
     opts.add_argument('--no-sandbox')
     opts.add_argument('--disable-dev-shm-usage')
+
     with webdriver.Chrome(options=opts) as driver:
         yield driver
 
+        driver.close()
```

Comment:
You shouldn't need to explicitly close the driver here, as the context manager should solve that for you (i.e. since you've specified this in a `with` block)